manifests/server.pp

```diff
@@ -132,6 +135,7 @@
 # @param include_dir You may include individual files or all files in a directory in the configuration file.
 # @param loadmodulepath Full path to location of server modules.
 # @param loadmodule Module to load at server startup.
+# @param sslcalocation_dir Location of certificate authority (CA) files for SSL server certificate verification.
```

Comment:
It would be better to open a new PR for the new parameter

Reply:
My guess is @bdeferme discovered the following parameters in `zabbix_server.conf` while converting to the new epp template:

- SSLCALocation
- VMwarePerfFrequency

As they were present in upstream config, and noticed they were missing in the old erb template, and added them to be consistent with upstream config.

@root-expert  I’ve deleted them as requested, but I think they should be included.  However, I won’t go out of my way to create a new PR if it is not required in my own production environment.